azurerm/internal/services/compute/windows_virtual_machine_resource.go

```diff
@@ -221,6 +221,12 @@ func resourceWindowsVirtualMachine() *schema.Resource {
 
 			"secret": windowsSecretSchema(),
 
+			"skip_shutdown": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
```

Comment:
(same here) can we move this to the `features` block since this is a behavioural feature toggle?